python/ray/train/tests/test_tune.py

```diff
@@ -325,21 +325,24 @@ def train_func():
 def test_tune_torch_get_device_gpu(ray_2_node_4_gpu, num_gpus_per_worker):
     from ray import tune
     from ray.tune.tuner import Tuner, TuneConfig
+    from ray.air.config import ScalingConfig
 
     num_samples = 2
 
     @patch("torch.cuda.is_available", lambda: True)
     def train_func():
         train.report(device_id=train.torch.get_device().index)
 
+    scaling_config = {
+        "num_workers": 2,
+        "use_gpu": True,
+        "resources_per_worker": {"GPU": num_gpus_per_worker},
+    }
+
     trainer = TorchTrainer(
         train_func,
         torch_config=TorchConfig(backend="gloo"),
-        scaling_config={
-            "num_workers": 2,
-            "use_gpu": True,
-            "resources_per_worker": {"GPU": num_gpus_per_worker},
-        },
+        scaling_config=ScalingConfig(*scaling_config),
```

Comment:
this change got merged in in the morning, can you pull in master?

Reply:
done.